docs/en/usage-with-symfony.md

```diff
@@ -0,0 +1,58 @@
+# Usage with Symfony
+
+Require migrations and DB driver. Let's use SQLite for this example:
+
+```
+composer require yiisoft/db-migration
+composer require yiisoft/db-sqlite
+```
+
+Configure migrations and database connection in your `config/services.yml`:
+
+```yaml
+Yiisoft\Db\Migration\:
```

Comment:
Remove trailing slashes?

Reply:
They're needed

Reply:
Sorry, overlooked that these are composite namespaces.